CONTRIBUTING.md

```diff
@@ -24,6 +24,18 @@ npx turbo run dev --filter example-basic...
 
 Finally, visit <http://localhost:3000/swingset> in your browser. Any changes to the core `swingset` package will be rebuilt and reflect in the running example app.
 
+## Debugging the example app
+
+If you are iterating on the loader code, it might be helpful to use the Node debugger instead of relying on `console.log()` statements. To make this easier, we have included a VSCode launch configuration.
```

Comment:
```suggestion
If you are iterating on the loader code, it might be helpful to use the Node.js [Debugger](https://nodejs.org/api/debugger.html) instead of relying on `console.log()` statements. To make this easier, we have included a VSCode launch configuration.
```